Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftRequestController.cs

```diff
@@ -142,136 +143,160 @@ public class OpenShiftRequestController : Controller
-                        {
-                            this.telemetryClient.TrackTrace($"{Resource.SubmitOpenShiftRequestToKronosAsync} - Operation to submit the DRAFT request has succeeded with: {postDraftOpenShiftRequestResult?.Status}");
-
-                            // Step 5 - Update the submitted Open Shift Request from DRAFT to SUBMITTED
-                            // so that it renders inside of the Kronos WFC Request Manager for final approval/decline.
-                            var postUpdateOpenShiftRequestStatusResult = await this.openShiftActivity.PostOpenShiftRequestStatusUpdateAsync(
-                                userMappingRecord.KronosPersonNumber,
-                                postDraftOpenShiftRequestResult.EmployeeRequestMgmt?.RequestItems?.EmployeeGlobalOpenShiftRequestItem?.Id,
-                                openShiftReqQueryDateSpan,
-                                Resource.KronosOpenShiftStatusUpdateToSubmittedMessage,
-                                new Uri(allRequiredConfigurations.WfmEndPoint),
-                                allRequiredConfigurations.KronosSession).ConfigureAwait(false);
-
-                            if (postUpdateOpenShiftRequestStatusResult?.Status == ApiConstants.Success)
+                            var responseContent = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                            graphOpenShift = JsonConvert.DeserializeObject<GraphOpenShift>(responseContent);
+
+                            // Logging the required Open Shift ID from the Graph call.
+                            this.telemetryClient.TrackTrace($"OpenShiftRequestController - OpenShift Graph API call succeeded with getting the Open Shift: {graphOpenShift?.Id}");
+
+                            var shiftStartDate = graphOpenShift.SharedOpenShift.StartDateTime.AddDays(
+                                                -Convert.ToInt16(this.appSettings.CorrectedDateSpanForOutboundCalls, CultureInfo.InvariantCulture))
+                                                .ToString(this.appSettings.KronosQueryDateSpanFormat, CultureInfo.InvariantCulture);
+
+                            var shiftEndDate = graphOpenShift.SharedOpenShift.EndDateTime.AddDays(
+                                               Convert.ToInt16(this.appSettings.CorrectedDateSpanForOutboundCalls, CultureInfo.InvariantCulture))
+                                               .ToString(this.appSettings.KronosQueryDateSpanFormat, CultureInfo.InvariantCulture);
+
+                            var openShiftReqQueryDateSpan = $"{shiftStartDate}-{shiftEndDate}";
+
+                            // Builds out the Open Shift Segments prior to actual object being built.
+                            // Take into account the activities of the open shift that has been retrieved
+                            // from the Graph API call made between lines 179-184.
```

Comment:
Line numbers in comments are brittle. Already this doesn't seem to make sense, there's no Graph call on lines 179-184.

Reply:
@aosolis , Removed line numbers from comments.